{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "2020-12-21T17:00:00-0400",
        "start": "2020-12-21T13:41:00-0400"
      },
      "geom": {
        "p": [
          "gttoGnxrbK"
        ]
      },
      "id": "4",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
